index.html

```diff
@@ -2035,9 +2030,37 @@ <h3>Securing Mechanisms</h3>
 mutually exclusive.
         </p>
         <p>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be secured by at least
-one securing method.
+<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be
+secured by at least one securing mechanism.
+        </p>
+        <p>
+Securing mechanism specifications:
         </p>
+
+        <ul>
+          <li>
+MUST document normative algorithms that can be used to secure and verify
+<a>verifiable credentials</a> and <a>verifiable presentations</a>.
+          </li>
+          <li>
+MUST provide a verification mechanism that returns only the information in the
+<a>conforming document</a> that has been secured and MAY provide additional
+information that might be helpful to developers. For the avoidance of doubt,
+the securing mechanism specification MUST NOT return any information that was
+not secured in the <a>conforming document</a>. The concrete interface that is
```

Comment:
This seems contradictory to returning additional helpful information. Maybe this:

```suggestion
MUST provide a verification mechanism that returns only the information in the
<a>conforming document</a> that has been secured. It MAY provide an
additional mechanism to receive other information that might be helpful
to developers. The concrete interface that is
```

This way additional information to help developers can be returned (as stated) via an opt-in and clearly specified mechanism.